package.json

```diff
@@ -49,7 +49,7 @@
       },
       {
         "repository": "https://github.com/foosbyte/kicker-trainer.git",
-        "branch": "next",
+        "branch": "update-node-and-hops",
```

Comment:
Ah wait, we have to remove this first 😬

Reply:
I actually thought about merging it with this change, then merging the kicker-trainer (while keeping the branch) & then revert this change in another PR.